bindings/c/test/mako/mako.c

```diff
@@ -1386,6 +1387,7 @@ void usage() {
 	printf("%-24s %s\n", "    --tpsmin=TPS", "Specify the target min TPS");
 	printf("%-24s %s\n", "    --tpsinterval=SEC", "Specify the TPS change interval (Default: 10 seconds)");
 	printf("%-24s %s\n", "    --tpschange=<sin|square|pulse>", "Specify the TPS change type (Default: sin)");
+	printf("%-24s %s\n", "    --stats-samplerate=RATE", "Specify the sampling rate for tps stats (Default: sample once per 1000 ms)");
```

Comment:
This is really not "sampling". This is just the print out frequency...

Reply:
It does seem to re-calculate ops, in `print_stats()` though, no?

Reply:
It's just computing TPS. The stats are always collected.